### ISSUE TITLE
Investigate duplicate css

### ISSUE THREAD

**Issue by furstenberg** [itslearning/protomorph#23]
A [recent MR from CIA](
https://gitlab.itsbuild.net/Itslearning/main/merge_requests/8327/diffs) had duplicate css from the Prometheus button component in their bundle css.

We need to investigate why this happens.

**Comment by plumpNation** [comment]
There should be 2 branches with 2 different techniques to solve this already in this repo. es-import-scss-once and import-scss-once. It's kind of nice to have the scss only version, but it puts extra code into your scss files that you may not want. I think I stole the idea from zurb. It was a while ago.

**Comment by furstenberg** [comment]
Yeah, I haven't had time to investigate further. There were duplicate scss for prom-button. 

But your branches are still there and it's on my todo-list to investigate.
